airgun/entities/http_proxy.py

```diff
@@ -0,0 +1,92 @@
+from navmazing import NavigateToSibling
+
+from airgun.entities.base import BaseEntity
+from airgun.navigation import NavigateStep, navigator
+from airgun.views.http_proxy import (
+    HTTPProxyCreateView,
+    HTTPProxyEditView,
+    HTTPProxyView,
+)
+
+
+class HTTPProxyEntity(BaseEntity):
+    endpoint_path = '/http_proxies'
+
+    def create(self, values):
+        """Create a new http-proxy."""
+        view = self.navigate_to(self, 'New')
+        view.fill(values)
+        view.submit.click()
+        view.validations.assert_no_errors()
+        view.flash.assert_no_error()
+        view.flash.dismiss()
+
+    def search(self, value):
+        """Search for 'value' and return http-proxy names that match.
+
+        :param value: text to filter (default: no filter)
+        """
+        view = self.navigate_to(self, 'All')
+        return view.search(value)
+
+    def read(self, entity_name, widget_names=None):
+        """Return dict with properties of http-proxy."""
+        view = self.navigate_to(self, 'Edit', entity_name=entity_name)
+        return view.read(widget_names=widget_names)
+
+    def update(self, entity_name, values):
+        """Update an existing http-proxy."""
+        view = self.navigate_to(self, 'Edit', entity_name=entity_name)
+        view.fill(values)
+        view.submit.click()
+        view.validations.assert_no_errors()
+        view.flash.assert_no_error()
+        view.flash.dismiss()
+
+    def delete(self, entity_name):
+        """Delete existing http-proxy entity"""
+        view = self.navigate_to(self, 'All')
+        self.search(entity_name)
+        view.table.row(Name=entity_name)['Actions'].widget.click(
+            handle_alert=True)
```

Comment:
where are you deleting here?

Reply:
There is only one button in Action column i.e "Delete". So above code works.
I can also use `view.table.row(name=entity_name)['Actions'].widget.fill('Delete')`

Reply:
There is only one button in Action column i.e. "Delete".